### PR TITLE
string.toUpperCase

### PR DESCRIPTION
**Pull request by Jacob-Tate**
Added string.toUpperCase as there is a project I'm working on that would benefit from it!

### CODE REVIEW

File: libs/pxt-common/pxt-helpers.ts [hunk 1]
```diff
@@ -402,6 +402,22 @@ namespace helpers {
...
+            if (97 <= c && c <= 122) {
+                r += s.slice(prev, i) + String.fromCharCode(c - 32)
+                prev = i + 1
+            }
+        }
+        r += s.slice(prev)
+        return r
+    }
+
     // TODO move to PXT
```
Comment (jwunderl) [inline libs/pxt-common/pxt-helpers.ts:421]:
```suggestion
```

File: libs/pxt-common/pxt-helpers.ts [hunk 2]
```diff
@@ -402,6 +402,22 @@ namespace helpers {
         return stringSubstrHelper(s, start, end - start);
     }
 
+    // TODO move to PXT
```
Comment (jwunderl) [inline libs/pxt-common/pxt-helpers.ts:405]:
```suggestion
```

I see this is duplicated from tolowercase / was accidentally left in when that was originally moved into pxt, but might as well drop the errant comments now ~

**Review by jwunderl — APPROVED** [review]
Looks good to me, thanks for the contribution!

**Review by riknoll — APPROVED** [review]
LGTM!

**Comment by jwunderl** [comment]
Thanks again for fixing this!

**Pull request merged**